src/attrs.rs

```diff
@@ -86,16 +86,25 @@ impl quote::ToTokens for Attrs {
     }
 }
 
+const MERMAID_INIT_SCRIPT: &str = r#"
+    var docs_theme = currentTheme.href;
```

Comment:
It is better to use `let` instead of `var` since it doesn't add garbage to global scope. Also I think it makes sense to somehow gracefully handle missing `currentThem`e.

Reply:
Done